__tests__/app/components/EditPassword.test.tsx

```diff
@@ -21,6 +21,8 @@ describe('EditPassword Component', () => {
         avatar: '',
         college: '',
         country: 'IN',
+        current_level: 1,
+        current_stars: 1,
```

Comment:
What happened to consistent variables names based on a universal style guide?
So you are allowed to use any number of naming conventions within a project now?